### DISCUSSION TITLE
MyJDownloader missing?

### DISCUSSION THREAD

**Discussion by mtoensing** [tteck/Proxmox#2479] [Ideas]
I was quite shocked to learn that we got a PMS LCX but not jDownloader LCX?

**Comment by chulojay** [comment]
bump i still use JD
 on my day to day.

**Comment by rcasula** [comment]
I would love it too!

An example could be found here https://github.com/StevenSeifried/proxmox-scripts and https://github.com/StevenSeifried/proxmox-scripts/issues/12#issuecomment-1356144716.
